jupyter_elfinder/elfinder.py

```diff
@@ -543,39 +537,34 @@ def __zipdl(self) -> None:
         pass
 
     def __file(self) -> None:
+        self._http_header["Content-type"] = "text/html"
         target = self._request.get(API_TARGET)
         if not target:
-            self._response[R_ERROR] = "Invalid parameters"
+            self._response["__text"] = "Invalid parameters"
```

Comment:
I'm saving the response text under this magic key. This allows us to keep the return value of the run method more simple in regards to typing.